docs/_documentations/troubleshooting.md

```diff
@@ -416,7 +416,48 @@ These steps reproduce the issue:
 6. Click **Finish**. The OpenAPI generator fails if the folder doesn't already exist.
 
 **Workaround:**
+For the VS Code extension: 
 - Manually create the output folder before you start the OpenAPI generator wizard. In the wizard, you can manually edit the **Output folder** text field. Ensure that the path points to a valid folder in the project.
 - Or you can use the **Browse...** button to select the folder that you want to use. The **Output folder** field is updated with the path.
 
-For post-client or post-server stub generation, use a separate output folder for code generation. Depending on the language and the generator type, the OpenAPI generator generates both source code files and build-related files. Some refactoring might be necessary. For example, if you are working with an existing Java or Maven project, move the generated source code to the proper source folder that already exists in the project. However, if your project is empty, the target output folder can be the root of the project, and you don't need to do as much refactoring and merging.
\ No newline at end of file
+For post-client or post-server stub generation, use a separate output folder for code generation. Depending on the language and the generator type, the OpenAPI generator generates both source code files and build-related files. Some refactoring might be necessary. For example, move the generated source code to the proper source folder that already exists in the project. However, if your project is empty, the target output folder can be the root of the project, and you don’t need to do as much refactoring and merging. 
+
+For Eclipse, for Java-based code generation, the OpenAPI generator generates code files and build-related files in the project's root folder. It also performs some automatic configuration, including pom.xml file merging.
```

Comment:
Can we change the new paragraph?  We want to convey the point that only Eclipse Java-based code generator wizard has some additional support.  Something like this:

For Eclipse, for Java-based code generators, the Open API wizards provide additional support to configure the project.   It is recommended that the project's root folder is selected as the output folder of the generator so that .java files will be generated into the existing `src/main/java` and `src/test/java` folders.  The wizard's default value of the output folder is the project's root folder.   The wizard also performs some automatic configuration, including pom.xml file merging, and necessary updates to the project's classpath.

Reply:
New paragraph changed to that suggested. Also removed the sentence at the beginning of this workaround according to your other comment.